content/docs/platform/quickstart/vue.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: 'Vue'
+pageTitle: 'Novu Vue Quickstart Guide'
+description: 'Create an account and learn how to start using Novu Inbox in your vue application.'
+---
+
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip';
+import {
+  Code2,
+  Library,
+  Workflow
+} from 'lucide-react';
+
+This guide walks you through integrating Novu’s Inbox into your Vue application for real time in-app notifications, from setup to triggering your first notification. By the end, you'll have a working notification inbox.
+
+<Callout type="info">
+  This guide uses @novu/js headless library to build the Inbox component in Vue. Novu currently does not support native Vue Inbox component.
+</Callout>
+
+<Steps>
+  <Step>
+  ### Create a Novu account
+    [Create a Novu account](https://dashboard-v2.novu.co/auth/sign-up) or [sign in](https://dashboard-v2.novu.co/auth/sign-in) to an existing account.
+  </Step>
+
+    <Step>
+  ### Create a Vue application
+    Create a new vue app with create-vue package. Skip this step if you already have an existing project:
+
+    ```bash
+    npm create vue@latest novu-inbox-vue
+    ```
+
+    Navigate to the newly created project:
+
+    ```bash
+    cd novu-inbox-vue
+    ```
+
+    ```bash
+    npm install
+    ```
+  </Step>
+
+  <Step>
+  ### Install the required package
+  Run the command below to install [Novu headless SDK](/platform/sdks/headless), which provides required component for Inbox UI:
+
+    ```bash
+    npm install @novu/js
+    ```
+  </Step>
+
+
+  <Step>
+  ### Add the Inbox component
+  Create the `src/components/NovuInbox.vue` file to add the Inbox component passing :tooltip[applicationIdentifier]{label="The application identifier is a unique identifier for your application. You can find it in the Novu Dashboard under the API keys page."} and :tooltip[subscriberId]{label="The subscriber ID is the unique identifier for the user in your application, typically the user's id in your database."}:
+
+```vue title="src/components/NovuInbox.vue"
+<template>
+  <div ref="novuInbox"></div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted, onUnmounted } from "vue";
+import { NovuUI } from "@novu/js/ui";
+
+interface NovuOptions {
+  options: {
+    applicationIdentifier: string;
+    subscriberId: string;
+  };
+}
+
+const novuInbox = ref<HTMLElement | null>(null);
+let novuInstance: NovuUI | null = null;
+
+onMounted(() => {
+  if (!novuInbox.value) {
+    console.error("Novu inbox container element not found");
+    return;
+  }
+
+  try {
+    const novu = new NovuUI({
+      options: {
+        applicationIdentifier: 'YOUR_APPLICATION_IDENTIFIER',
+        subscriberId: 'YOUR_SUBSCRIBER_ID',
+      },
+    } as NovuOptions);
+
+    novu.mountComponent({
+      name: "Inbox",
+      props: {},
+      element: novuInbox.value,
+    });
```

Comment:
I would suggest to explain what happens here, where we mount the component on top of some vue div referecne